test/unit/captureTasksController.spec.js

```diff
@@ -0,0 +1,54 @@
+describe('CaptureTasksController', function(){
+  beforeEach(module('Capture'));
+
+  var ctrl;
+
+  beforeEach(inject(function($controller){
+    ctrl = $controller('CaptureTasksController');
+  }));
+
+  it('initialises with an empty task load', function(){
+    expect(ctrl.loadTask).toBeUndefined();
+  });
+
+  it('can add a task to the start of the list', function(){
+    ctrl.addTask('A first Task');
+    ctrl.addTask('A second Task');
+    expect(ctrl.tasks[0].desc).toEqual('A second Task')
+  });
+
+  it('can date stamp a task', function(){
+    ctrl.addTask('A new Task');
+    expect(ctrl.tasks[0].createdAt).toEqual(Date.now())
```

Comment:
When testing date stamps, you need to figure out a way to freeze the date - if the task takes a long time to finish, the value of `Date.now()` will have moved on by a few milliseconds